pkg/acceptance/compose_test.go

```diff
@@ -31,6 +32,7 @@ func TestComposeGSS(t *testing.T) {
 }
 
 func TestComposeGSSPython(t *testing.T) {
+	skip.WithIssue(t, 81254)
```

Comment:
we want to remove this backport (revert a419dff5aebbc62e34cbd47bd898d3e4492f4705)

Reply:
Thanks! Ack, I'll revert this.

It seems that `git merge` is having trouble understanding that a commit in both branches was then superseded by a subsequent commit in just the release-22.1 branch. 

In doing an audit, it looks like all release-22.1.0 backports have a corresponding release-22.1 backports, except for this:

* https://github.com/cockroachdb/cockroach/pull/80574

I'll update this merge commit accordingly.